frontstage/controllers/party_controller.py

```diff
@@ -233,6 +233,15 @@ def get_respondent_enrolments(party_id):
                 }
 
 
+def get_respondent_enrolments_for_known_collex(enrolment_data, cache_collex):
+    """remove any enrolments that are for surveys not in already started collex."""
```

Comment:
Sorry to be picky, but the Why here is important.  Mentioning the problem we're fixing will clue future people as to why it's in here